.husky/pre-commit

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
+# Check for .only in the staged changes of test files
+if git diff --cached | grep -E '^\+\s*(test|it|describe)\.only'; then
```

Comment:
I think the CI will be detected but just in case

```suggestion
if git --no-pager diff --cached | grep -E '^\+\s*(test|it|describe)\.only'; then
```

Reply:
oh, good catch, thanks!